analytics-api/src/main/java/org/exoplatform/analytics/model/filter/aggregation/AnalyticsAggregation.java

```diff
@@ -87,6 +90,19 @@ public AnalyticsAggregation(String field) {
     this.type = AnalyticsAggregationType.TERMS;
   }
 
+  public AnalyticsAggregation(AnalyticsAggregationType type, String field, String sortDirection, String interval, String offset, long limit, boolean useBounds, long minBound, long maxBound) {
+    super();
+    this.type = type;
+    this.field = field;
+    this.sortDirection = sortDirection;
+    this.interval = interval;
+    this.limit = limit;
+    this.offset = offset;
+    this.useBounds = useBounds;
+    this.minBound = minBound;
+    this.maxBound = maxBound;
+  }
+
```

Comment:
no needed IMO

Reply:
this is a constructor with not all  args